bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/values/OpenCloseValue.java

```diff
@@ -22,34 +26,60 @@
 import org.openhab.core.types.Command;
 
 /**
- * Implements an open/close boolean value.
+ * Implements an Open/Closed boolean value.
  *
  * @author David Graeff - Initial contribution
  */
 @NonNullByDefault
 public class OpenCloseValue extends Value {
-    private final String openString;
-    private final String closeString;
+    private final Set<String> openStates;
+    private final Set<String> closedStates;
+    private final String openCommand;
```

Comment:
Is it possible to somehow send a command to an OpenCloseValue? You didn't introduce that in this PR, but you definitely extend it. Assuming it's not possible, I'd rather just remove it (possibly throwing an exception in getMQTTpublishValue and parseCommand, and renaming the current parseCommand to parseMessage - though if we go that way, you need to make sure to maintain the UnDefType condition from the parent class)

Reply:
Normally it's not possible to send commands to Contact Items, so I guess it's unlikely a command is sent here, however I am not sure if this code is possibly needed to "publish" a contact Item state to MQTT.